GoMeet/src/main/webapp/meeting-event.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>GoMeet</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1 style="text-align:center;">GoMeet</h1>
```

Comment:
nit: styling should be put into the style sheet: https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

Reply:
Fixed :)